lib/open_api_spex/operation_dsl.ex

```diff
@@ -0,0 +1,57 @@
+defmodule OpenApiSpex.OperationDsl do
```

Comment:
This may still need a `__using__` macro to inject the `open_api_operation(action)` function into the controller?
I like the way that having a callback between the framework and user code allows the operation DSL to be changed or customized.

Reply:
Indeed, this will need to automatically define `operation_api_operation(action)` for each action. Commit 4e1e1e9b019d23dd1cbddc0e52e4b8f594758cca does that now. It still does this without leveraging the `__using__` macro. If we can avoid requiring a call to `use ...`, that may make a slightly better API.